src/ai_hawk/llm/llm_manager.py

```diff
@@ -561,13 +566,14 @@ def answer_question_textual_wide_range(self, question: str) -> str:
 
         if section_name == "cover_letter":
             chain = chains.get(section_name)
-            output = chain.invoke(
+            raw_output = chain.invoke(
                 {
                     RESUME: self.resume,
                     JOB_DESCRIPTION: self.job_description,
                     COMPANY: self.job.company,
                 }
             )
+            output = self._clean_llm_output(raw_output)
```

Comment:
I would suggest rather than cleanig forcing LLM is better option. (prompt engineering)